modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala

```diff
@@ -326,4 +326,35 @@ trait RunScriptTestDefinitions { _: RunTestDefinitions =>
         expect(p.out.trim() == "List(1, 2, 3, -v)")
       }
     }
+
+  test("script file with shebang header and no extension run with scala-cli shebang") {
```

Comment:
Both tests seem to fail on Windows, as it doesn't have shebang support by default (unless with `cygwin`, but I don't think it's available on our CI). I think we can simply skip them on Windows.

```suggestion
  if (!Properties.isWin) {
    test("script file with shebang header and no extension run with scala-cli shebang") {
```